HelloMixpanel/HelloMixpanel/HelloMixpanel-Prefix.pch

```diff
@@ -1,7 +0,0 @@
-//
```

Comment:
Did you try running the tests? I think they were failing because this file was missing.